django_netjsonconfig/tests/test_admin.py

```diff
@@ -8,6 +10,8 @@
 from ..models import Config, Device, Template, Vpn
 from . import CreateConfigMixin, CreateTemplateMixin, TestVpnX509Mixin
 
+devnull = open(os.devnull, 'w')
```

Comment:
does this really need to be opened with `w`?

Can you add a `tearDownClass` classmethod which closes the file please?

Reply:
>##### git rebase -i HEAD~8
>
Result is:
`fatal: Needed a single revision`
`invalid upstream 'HEAD-8`
Since I have never done squashing priorily, I Googled it to find another command
##### git rebase master
>
This command does open up a Text editor but with  only one word `noop`, rest all is commented out.
Can you please help me out with this, I am extremely sorry to trouble you.

Reply:
> Can you add a tearDownClass classmethod which closes the file please?
> 
Done :smile:

Reply:
Please google how to do rebase interactive to squash commits